### PR TITLE
Fixed the stat64() syscall for QL_ARCH.MIPS

### PR DESCRIPTION
**Pull request by kr1tzy**
### Which kind of PR do you create?

- [x] This PR only contains minor fixes.
- [ ] This PR contains major feature update.
- [ ] This PR introduces a new function/api for Qiling Framework.

### Coding convention?

- [x] The new code conforms to Qiling Framework naming convention.
- [x] The imports are arranged properly.
- [x] Essential comments are added.
- [x] The reference of the new code is pointed out.

### Extra tests?

- [x] No extra tests are needed for this PR.
- [ ] I have added enough tests for this PR.
- [ ] Tests will be added after some discussion and review.

### Changelog?

- [ ] This PR doesn't need to update Changelog.
- [x] Changelog will be updated after some proper review.
- [ ] Changelog has been updated in my PR.

### Target branch?

- [x] The target branch is dev branch.

### One last thing

- [x] I have read the [contribution guide](https://docs.qiling.io/en/latest/contribution/)

-----

### CODE REVIEW

**Comment by xwings** [comment]
Hi,

Welcome to the community

Can you please solve the conflict and i will merge

thanks!

File: qiling/os/posix/syscall/stat.py [hunk 1]
```diff
@@ -163,102 +163,109 @@ def create_stat64_struct(ql, info):
...
+        stat64_buf += ql.pack64(0)
+        stat64_buf += ql.pack64(info.st_size)
+        stat64_buf += ql.pack64(info.st_blksize)  # ?? stat_info.st_blksize
+        stat64_buf += ql.pack64(info.st_blocks)  # ?? stat_info.st_blocks
+        stat64_buf += ql.pack64(int(info.st_atime))
+        stat64_buf += ql.pack64(int(info.st_mtime))
+        stat64_buf += ql.pack64(int(info.st_ctime))
+        stat64_buf += ql.pack64(info.st_ino)
     elif ql.ostype == QL_OS.MACOS:
-        fstat64_buf = ql.pack32(0x0)              # st_dev            32byte
```
Comment (xwings) [inline qiling/os/posix/syscall/stat.py:210]: macos 10.15, we need some hack to make this st_dev to 0x0

can you please add it back

Comment (kr1tzy) [reply qiling/os/posix/syscall/stat.py:210]: Done

**Comment by xwings** [comment]
Fix break.

Thanks for the PR

**Pull request merged**